### ISSUE TITLE
VFS: Never expiring dir cache

### ISSUE THREAD

**Issue by toomuchio** [rclone/rclone#4476]
Would be nice if the --dir-cache-time option allowed for the cache to never expire, you can always invalidate it manually if it falls out of sync because the mount was down and it missed some polls.

But in my case I only make changes to the source via rclone mount. So caching forever would be great and less load on my cloud provider.

**Comment by animosity22** [comment]
You can do that just by setting to a very large value.

**Comment by toomuchio** [comment]
Yes, you can do that, that's what I do now... But having the option to indefinitely set it perhaps passing 0 or -1 is what I'm asking for.

**Comment by animosity22** [comment]
Just seems like a bit of a waste of effort as the functionality is already there with setting a value high.

**Comment by ncw** [comment]
Actually I think you can pass "off" already. This sets the duration to be very large about 500 years.

I don't think this is documented...

**Comment by toomuchio** [comment]
It doesn't appear to be documented is there an easy way I can test? The code just seems to forward to the time parser don't see anything doing an == 'off' override to 500 years.

**Comment by ncw** [comment]
You can run with --rc and query the options that will show you the parsed value
